Source/NSView.m

```diff
@@ -5086,6 +5086,30 @@ - (void) bind: (NSString *)binding
     }
 }
 
+- (NSViewLayerContentsPlacement) layerContentsPlacement
+{
+  // FIXME (when views have CALayer support)
+  return NSViewLayerContentsPlacementScaleAxesIndependently;
```

Comment:
In situations like these, I am also never sure whether to do a one-off `NSLog` to warn the developer that this implementation of `-[NSView setLayerContentsPlacement:]` is a stub. Maybe it makes sense here, given that this getter, if it weren't already, would make the property setter a no-op?

So both for the getter and the setter:

```
static BOOL logged = NO;
if (!logged)
  {
    NSLog("warning: stub no-op implementation of -[NSView setLayerContentsPlacement:]");
    logged = YES;
  }
```

(Is there a wrapper around NSLog for warnings? I don't recall.)

Whether that makes sense -- I'm leaving that to you, I'm not going to insist on it :)